src/app/fyle/personal-cards/personal-cards.page.ts

```
@@ -659,7 +686,7 @@ export class PersonalCardsPage implements OnInit, AfterViewInit {
       or: [],
     };
     newQueryParams.btxn_status = `in.(${this.selectedTransactionType})`;
-    newQueryParams.ba_id = 'eq.' + this.selectedAccount;
+    newQueryParams.ba_id = 'eq.' + this.selectedAccount.id;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)